src/Nethermind/Nethermind.Optimism/OptimismPlugin.cs

```diff
@@ -177,6 +177,16 @@ public Task InitSynchronization()
             _api.LogManager
         );
 
+        PivotUpdator pivotUpdator = new(
```

Comment:
Variable is not used anywhere. Is it constructor that does something useful here?

Reply:
Yep, it starts a loop inside, looks weird 100%. But without assignment it even less clear imho

Reply:
Won't `_ = new PivotUpdator(...` look more straightforward?